src/infra/main/template.yaml

```diff
@@ -151,3 +151,140 @@ Resources:
               - Fn::ImportValue: !Sub "${CoreStackName}-AccountStatusTableArn"
       Roles:
         - !Ref AISSampleLambdaHandlerRole
+
+ #### TxMA ####
+  TxMAIngressEncryptionKey:
```

Comment:
can we call this key something more general like "SQSEncryptionKey" like in vc storage? as we are likely to reuse it for other queues